labs-ios-starter/Application/Extensions/String + identifier.swift

```diff
@@ -24,7 +24,16 @@ enum TableViewIdentifier: String {
     case profileCell
 }
 
+enum CollectionViewIdentifier: String {
```

Comment:
Could also refactor this to be, `CellReuseIdentifier` if you found that to be more specific